app/contributing/index.md

```diff
@@ -4,53 +4,42 @@ no_version: true
 layout: 'docs-v2'
 ---
 
-Hello, and welcome! Thanks for thinking about contributing to the Kong documentation.
-
-This section of the docs is here to help you help us, so read on to learn how to ask questions and help effectively.
-We respect your time, and the more you follow our guidelines the easier it is for us (docs maintainers) to respond
-promptly and help you get your pull requests merged.
+Hello, and welcome! Thanks for your interest in contributing to the Kong docs. The following information will help you contribute effectively, which will in turn help us to get your pull requests merged faster. 
 
 ## What we're looking for
 
 We welcome fixes to unclear prose, fixes to typos in docs for recent versions, docs for new features you've contributed to
 the code, and more.
 
-If you've written a Kong plugin and need to contribute documentation for it, see the docs [about plugin docs](/contributing/plugin-docs).
-There are special guidelines for these docs.
+We are currently accepting limited plugin submissions from trusted technical partners to our plugin hub. To learn more about our partner program, see our [Kong Partners page](https://konghq.com/partners/).
 
-We ask that you explore the existing documentation before you start a big docs contribution. Some types of docs
-don't belong on the site: end-to-end guides, tutorials, anything better suited to a blog post. If you're interested in
-this kind of content, though, join the community on [Kong's forum](https://discuss.konghq.com/), on
+Explore the existing documentation before you start a big docs contribution. Some types of docs don't belong on the site, including: end-to-end guides, tutorials, and anything better suited to a blog post. If you're interested in
+this kind of content or have support questions, join the community on [Kong's forum](https://discuss.konghq.com/), on
 [Gitter](https://gitter.im/Kong/kong), or on IRC at #kong.
 
-The community is the place to ask support questions, too. We can't help you with the product in this repository.
-
-For bugs against Kong Gateway functionality, see the [code repository](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-report-bugs).
+To report Kong Gateway bugs, visit the [Kong product contributing guide](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-report-bugs).
```

Comment:
```suggestion
We can't provide product support in this repository. To report Kong Gateway bugs, visit the [Kong product contributing guide](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-report-bugs).
```
This bit of info is really important, as we get people reaching out to the docs team about product issues occasionally, and want to be very clear that this isn't the place.